Kmozart/free-mentors
# Merge pull request #43 from Kmozart/ft-user-can-create-mentor-session…

#### What does this PR do?
Has CREATE of CRUD on API Database

#### Description of Task to be completed?
- Have the following endpoint working:
```POST/api/v1/sessions```

#### How should this be manually tested?
- ```git-clone``` this repo, cd into it and npm run dev then on Postman, check the above endpoint by clicking SEND. It should return a status 201 and a ```"session created!"``` message.

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- #168372417 Users can create mentor sessions - DB endpoint
#### Screenshots (if appropriate)
- N/A
#### Questions:
- N/A
